src/newdb.cpp

```diff
@@ -1134,8 +1233,10 @@ static bool save_char(char_data *player, DBIndex::vnum_t loadroom)
     }
   }
 
-  /* Figure out their vehicle-- they can only load in it if they own it. */
-  if (player->in_veh && player->in_veh->owner == GET_IDNUM(player))
+  /* Figure out their vehicle-- they can only load in it if they own it.  Unless we're calling from copyover.*/
+  if (player->in_veh && fromCopyover)
```

Comment:
This is probably the better way to format this check for readability:
```C
if (player->in_veh && (fromCopyover || player->in_veh->owner == GET_IDNUM(player)))
  inveh = player->in_veh->idnum;
```

Reply:
Done